.golangci.yml

```diff
@@ -0,0 +1,34 @@
+run:
+  deadline: 5m
+  skip-files:
+  - ".*gen_elements.*\\.go$"
```

Comment:
This is wrong. [`gen_elements.go`](https://github.com/loadimpact/k6/blob/master/js/modules/k6/html/gen/gen_elements.go) is the hand-written file and [`elements_gen_test.go`](https://github.com/loadimpact/k6/blob/master/js/modules/k6/html/elements_gen_test.go) is also a hand-written test file. The only generated file is [`elements_gen.go`](https://github.com/loadimpact/k6/blob/master/js/modules/k6/html/elements_gen.go) :smile: And even then, I don't think we should exclude *any* of these files from the linter. 

The only files we should exclude are the rice-box ones, since they're just static embedded content. The rest are actual Go code, and it doesn't matter if they're generated or not, they shouldn't have linter issues unless there's a very good reason. And for those with good reasons, `//nolint` should suffice

Reply:
adding `//nolint` to a generated file will be hard ... especially if we are not the ones generating it which is not the case. On the other hand golangci-lint automatically removes all autogenerated files it detects ... which obviously doesn't work in this case :)